src/index.ts

```diff
@@ -1,3 +1,6 @@
+// Default App
+export { default as App } from "./App";
```

Comment:
I thought about this last night. If we make this a default export, and someone wants to _only_ use one or two of our components, will they be forced to import the full App as well (thereby importing all the components)?

Reply:
If we make this a default export, users will have the flexibility to import whatever they want. So someone could choose to write `import {Footer, MeetingCard} from @cdp/frontend;` and then the `App` would never be downloaded.

Comparitively, an install with `App` and the components would look like so, `import App, {Footer, MeetingCard} from @cdp/frontend;`.

Nowwww with all that said, the line you highlighted is actually a named export for App (the curly braces mean named export) because I couldn't quite figure out how to make App a default export from src/index.ts and wanted to be able to move on and work on `App`. So in reality someone importing will write this:

`import {App} from @cdp/frontend;`

Or this if they want more components:

`import {App, Footer, MeetingCard} from @cdp/frontend;`

Reply:
Weird. I think I am partial to making it a non-default export for some reason.

Crazy that you can have two export patterns in a single line.

Reply:
Haha yeah I know I was very intrigued when I saw that pattern for the first time.

Also sounds good I'll just won't change anything and keep App as a named export for now then.